app/services/staff_action_logger.rb

```diff
@@ -583,6 +583,30 @@ def log_post_rejected(rejected_post, opts = {})
     ))
   end
 
+  def log_web_hook(web_hook, action, opts = {})
+    details = [
+      "webhook_id: #{web_hook.id}",
+      "payload_url: #{web_hook.payload_url}"
+    ]
+
+    if changes = opts[:changes]
+      changes.reject! { |k,v| k == "updated_at" }
```

Comment:
You're missing a space 😉 

```suggestion
      changes.reject! { |k, v| k == "updated_at" }
```

Reply:
ugh 😄